src/status_im/contexts/wallet/send/events_test.cljs

```diff
@@ -163,3 +181,907 @@
+  (is (contains? (get-in (dispatch [event-id]) [:db :wallet :ui :send]) :other-props)))
+
+(h/deftest-event :wallet/clean-send-amount
+  [event-id dispatch]
+  (reset! rf-db/app-db
+    {:wallet {:ui {:send {:amount      10
+                          :other-props :value}}}})
+  (is (not (contains? (get-in (dispatch [event-id]) [:db :wallet :ui :send]) :amount)))
+  (is (contains? (get-in (dispatch [event-id]) [:db :wallet :ui :send]) :other-props)))
+
+(h/deftest-event :wallet/clean-disabled-from-networks
+  [event-id dispatch]
+  (reset! rf-db/app-db
+    {:wallet {:ui {:send {:disabled-from-chain-ids [:optimism]
+                          :other-props             :value}}}})
+  (is (not (contains? (get-in (dispatch [event-id]) [:db :wallet :ui :send]) :disabled-from-chain-ids)))
+  (is (contains? (get-in (dispatch [event-id]) [:db :wallet :ui :send]) :other-props)))
+
+(h/deftest-event :wallet/clean-from-locked-amounts
+  [event-id dispatch]
+  (reset! rf-db/app-db
+    {:wallet {:ui {:send {:from-locked-amounts "value"
+                          :other-props         :value}}}})
+  (is (not (contains? (get-in (dispatch [event-id]) [:db :wallet :ui :send]) :from-locked-amounts)))
+  (is (contains? (get-in (dispatch [event-id]) [:db :wallet :ui :send]) :other-props)))
+
+(h/deftest-event :wallet/disable-from-networks
+  [event-id dispatch]
+  (let [expected-db {:wallet {:ui {:send {:disabled-from-chain-ids [:optimism]
+                                          :other-props             :value}}}}]
+    (reset! rf-db/app-db
+      {:wallet {:ui {:send {:other-props :value}}}})
+    (is (match? expected-db (:db (dispatch [event-id [:optimism]]))))))
+
+(h/deftest-event :wallet/unlock-from-amount
+  [event-id dispatch]
+  (let [expected-db {:wallet {:ui {:send {:other-props         :value
+                                          :from-locked-amounts {}}}}}]
+    (reset! rf-db/app-db
+      {:wallet {:ui {:send {:other-props         :value
+                            :from-locked-amounts {:chain-id [1 10]}}}}})
+    (is (match? expected-db (:db (dispatch [event-id :chain-id]))))))
+
+(h/deftest-event :wallet/lock-from-amount
+  [event-id dispatch]
+  (let [expected-db {:wallet {:ui {:send {:other-props         :value
+                                          :from-locked-amounts {:amount 10}}}}}]
+    (reset! rf-db/app-db
+      {:wallet {:ui {:send {:other-props :value}}}})
+    (is (match? expected-db (:db (dispatch [event-id :amount 10]))))))
+
+(h/deftest-event :wallet/clean-selected-token
+  [event-id dispatch]
+  (reset! rf-db/app-db
+    {:wallet {:ui {:send {:other-props        :value
+                          :token              "ETH"
+                          :token-display-name "ETH"
+                          :tx-type            :tx/collectible-erc-721}}}})
+  (is (not (contains? (get-in (dispatch [event-id]) [:db :wallet :ui :send]) :token)))
```

Comment:
I think we should at least remove the obvious duplication from the tests in this file. There's no need to repeatedly `dispatch` and `get-in` the same thing. Like so:

```clojure
(h/deftest-event :wallet/clean-selected-collectible
  [event-id dispatch]
  (reset! rf-db/app-db
    {:wallet {:ui {:send {:other-props        :value
                          :collectible        "ETH"
                          :token-display-name "ETH"
                          :amount             10}}}})
  (let [send (get-in (dispatch [event-id]) [:db :wallet :ui :send])]
    (is (not (contains? send :collectible)))
    (is (not (contains? send :token-display-name)))
    (is (not (contains? send :amount)))
    (is (contains? send :other-props))))
```

Better yet I think would be to replace all these assertions with only one that checks the output of the function, but for that to work you would need to use this branch which I just opened https://github.com/status-im/status-mobile/pull/20825, but up to you if you want to wait until that one is merged, probably better to get this PR in `develop` first.

```clojure
(h/deftest-event :wallet/clean-selected-collectible
  [event-id dispatch]
  (reset! rf-db/app-db
    {:wallet {:ui {:send {:other-props        :value
                          :collectible        "ETH"
                          :token-display-name "ETH"
                          :amount             10}}}})
  (is (match-strict? {:db {:wallet {:ui {:send {:other-props :value}}}}}
                     (dispatch [event-id]))))
```